assignment/assignment.hs

```diff
@@ -52,8 +58,12 @@ count element list = 0 -- replace me
 -- help to write it as two functions. (Hint: you only
 -- need to traverse the list once)
 
-evensEqualOdds :: [ Integer ] -> Bool
-evensEqualOdds numbers = False -- replace me
+evensEqualOdds :: [ Int ] -> Bool
+evensEqualOdds numbers = if(alternateSumList numbers == 0) then True else False
+
+alternateSumList :: [ Int ] -> Int
+alternateSumList [] = 0
+alternateSumList (hd : tl : rest) = hd - tl + alternateSumList rest
 
```

Comment:
That is a really nice way of doing it: clean and concise. My only (very minor) observation is that `tl` normally means `tail`, the rest of the list...